datahub-web-react/src/app/entity/shared/components/styled/StripMarkdownText.tsx

```diff
@@ -5,9 +5,11 @@ import styled from 'styled-components';
 const RemoveMarkdownContainer = styled.div`
     display: block;
     overflow-wrap: break-word;
-    word-wrap: break-word;
-    overflow-x: hidden;
-    overflow-y: auto;
+    overflow: hidden auto;
+    white-space: nowrap;
+    width: 844px;
```

Comment:
okay so here we're actually going to be in a very similar situation like we talked about with setting a fixed `height` - if a our user has a slightly smaller screen/resolution, this fixed width may be too large and therefore we will have a horizontal scroll like we had before.

Instead I think we can make a couple of css additions in order to make this as responsive as we can. In order to make it responsive, we are going to need to set percentage widths (or utilize flex-box) where we ultimately need a fixed width higher up in the component tree.

In this instance, I would start by changing your `width: 844px;` -> `max-width: 100%;` - now that won't be enough since the `100%` is going to be based on this element's parent. So we need to propagate this up until we can safely set a fixed with.

Next I would set `max-width: 100%;` on `DescriptionContainer` in `DefaultPreviewCard.tsx`.

Then the final change I think we need is to set `max-width: calc(100% - 260px);` on `ResultContainer` in `SearchResults.tsx`.

I feel more comfortable setting that max-width with a static `260px` because we explicitly define the width of the `FiltersContainer` right next to it to be `260px` (I would even pull this 260 into a constant so they can all share it in case that number ever changes)

Let me know if all this makes sense!

Reply:
Sure, the updated changes are added to the PR. 
Thanks for highlighting the responsiveness issue for the fixed width